src/NuGet.Clients/NuGet.Console/PowerConsoleToolWindow.cs

```diff
@@ -472,15 +474,19 @@ private void StartConsoleSession(FrameworkElement consolePane)
                 {
                     if (WpfConsole.Dispatcher.IsStartCompleted)
                     {
-                        OnDispatcherStartCompleted();
+                        NuGetUIThreadHelper.JoinableTaskFactory.Run(async delegate
```

Comment:
You don't really need the extra delegate: 

```cs
                        NuGetUIThreadHelper.JoinableTaskFactory.Run(OnDispatcherStartCompletedAsync);
``` 

The above should do the job as well.

Reply:
Ok. Addressed.